packages/ui/src/theme-native/theme.tsx

```diff
@@ -27,8 +27,7 @@ export function generateTheme(colorScheme: ColorSchemeName) {
       round: 9999,
     },
     textVariants,
-    // Temporary workaround for type errors. TODO: Add a zIndex scale.
-    zIndices: undefined,
+    zIndices: zIndices,
```

Comment:
Maybe you can change this to `zIndices,` in future?

Reply:
Yeah, noticed it right after merging too.